.github/workflows/checkSite.yml

```diff
@@ -35,6 +35,11 @@ jobs:
         working-directory: href-checker
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
+      - name: Generate Sidebar
+        run: make site-gen-sidebar
+      - name: Normalize Book Introduction Pages
+        working-directory: site
+        run: find . -iname "00.md" -execdir mv {} README.md \; && sed -i s/00.md//g sidebar.md 
```

Comment:
this kind of processing should be encapsulated in the make target. Otherwise, you have leaky abstractions.

Reply:
Consolidated into the make target.